mlflow/ml-package-versions.yml

```diff
@@ -655,22 +655,21 @@ langchain:
           "transformers",
           "torch",
           "torchvision",
-          # TODO: The LLM saving/loading logic does not handle partner packages such as langchain-openai
-          # and always load llms from the legacy community package. Particularly for OpenAI LLM,
-          # the legacy version is not comptible with OpenAI SDK >= 1.8.0, so we need to pin the
-          # version here until resolving the loading issue.
-          "openai<1.8.0",
```

Comment:
note: This is resolved somehow (probably some recent update in community package), so now we can test with the latest OpenAI SDK:)

Reply:
Christmas comes early for our CI :)